pro/config/env.js

```diff
@@ -82,7 +83,8 @@ function getClientEnvironment(publicUrl) {
         SENTRY_SAMPLE_RATE: process.env.SENTRY_SAMPLE_RATE,
         SENTRY_SERVER_URL: process.env.SENTRY_SERVER_URL,
         URL_FOR_MAINTENANCE: process.env.URL_FOR_MAINTENANCE,
-        MAINTENANCE_PAGE_AVAILABLE: process.env.MAINTENANCE_PAGE_AVAILABLE === 'true' || false,
+        MAINTENANCE_PAGE_AVAILABLE:
+          process.env.MAINTENANCE_PAGE_AVAILABLE === 'true' || false,
```

Comment:
ne peut-on pas juste écrire : 

```suggestion
          process.env.MAINTENANCE_PAGE_AVAILABLE === 'true'
```

?

Reply:
si j'ai pas pensé à le faire (mon linter est juste passé sur cette ligne). Je rajoute ça